Week7/WEEK 7 SOLUTION.txt

```diff
@@ -0,0 +1,92 @@
+#Create CSV file for product selling for 6 Months ( Prod_No | Prod_Name | Jan | Feb | Mar | Apr | May | Jun) for 5 products.
+#Perform following operations.
+#1. Add 12 Records. Take input from user.
+
+import csv
+import pandas as pd
+
+header = ["Prod_No", "Prod_Name", "Jan", "Feb", "Mar", "Apr", "May", "Jun"]
+data=[[1,'PEN',100,200,200,350,200,300],[2,'PENCIL',200,400,400,350,200,300],[3,'ERASER',10,10,20,35,20,10],[4,'SHARPNER',100,200,200,350,200,300],[5,'SCALE',1000,2000,200,350,2000,3000]]
+
+with open('c:\sqlite3\csv\week7.csv','w',newline='') as f:
+    w=csv.writer(f)
+    w.writerow(header)
+    w.writerows(data)
+    for i in range(12):
+        no=int(input("Enter Product No. :"))
+        name=input("Enter Product Name :")
+        jan=int(input("Enter January sell :"))
+        feb=int(input("Enter February sell :"))
+        mar=int(input("Enter March sell :"))
+        apr=int(input("Enter April sell :"))
+        may=int(input("Enter May sell :"))
+        june=int(input("Enter June sell :"))
+        sub_lis=[no,name,jan,feb,mar,apr,may,june]
+        w.writerow(sub_lis)
+
+with open('c:\sqlite3\csv\week7.csv','r',errors='ignore') as f:
+    w=csv.reader(f)
+    for i in w:
+        print(i)
+
+#2. Create dataframe.
+csv_file='c:\sqlite3\csv\week7.csv'
+df=pd.read_csv(csv_file)
+df
+
+#3. Change Column Name Product No, Product Name, January, February, March, April, May, June.
+df.columns=["Product No","Product Name","January","February","March","April","May","June"]
+df
+
+#4. Add column "Total Sell" to count total of all month and "Average Sell"
+df["Total Sell"]=df.iloc[:,2:].sum(axis=1)
+df["Average Sell"]=df.iloc[:,2:8].mean(axis=1)
+df
+
+#5. Add 2 row at the end.
+df.loc[17]=(18,"NOTEBOOK",100,200,300,400,500,600,2100,350.000000)

```

Comment:
Here what if the user don't know the last index of dataframe..? Is there any solution for that..?